GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -143,109 +228,162 @@ void Renderer::renderCamera(std::shared_ptr<Scene> scene,
             renderPostEffectData(texture_render_texture, post_effects.back(),
                     post_effect_shader_manager);
         }
-    } // flag checking
-}
 
-void Renderer::renderCamera(std::shared_ptr<Scene> scene,
```

Comment:
did you really remove renderCamera?  or is diff just confused?  or are you not rebased to the monoscopic check-in?